Jenkinsfile

```diff
@@ -1,17 +1,17 @@
 pipeline {
   environment {
-            GENERATOR_IMAGE = 'quay.io/ocpmetal/assisted-ignition-generator'
+            GENERATOR = 'quay.io/ocpmetal/assisted-ignition-generator'
```

Comment:
maybe rename to GENERATOR_IMAGE_NAME ( just a thought..)? So it will be more clearer